util/topgen.py

```diff
@@ -997,6 +1014,16 @@ def main():
         default=False,
         action="store_true",
         help="If set, the tool generates top level RAL model for DV")
+    parser.add_argument(
```

Comment:
i.e. the `ral` entry would become:
```
  ral:
    generator: ralgen
    parameters:
      name: chip
      top_hjson: ../../data/top_earlgrey.hjson
      alias_hjson:
        - ../../../foundry/[...]/prim_<tech-node>_otp.hjson
        - ../../../foundry/[...]/prim_<tech-node>_flash.hjson
```

Reply:
sounds good. Can you remind me, does that mean we need to expose the existence of foundry in the open?
i might have missed it, but it might makes sense to document somewhere how in general this should be utilized. Even if it's just a couple of slide deck for the silicon wg.

Reply:
No, if I did this correctly we do not have to add this on the opensource side. 
We would place this in the closed source core file.

Once all this landed, I plan to update the (relatively stale) documentation update that has been pending here: https://github.com/lowRISC/opentitan/pull/8571
I will also give an overview with a few slides in the silicon WG so that everyone knows about this mechanism.